ol-keycloak/ol-spi/src/main/java/edu/mit/keycloak/util/OpenAttribute.java

```diff
@@ -0,0 +1,5 @@
+package edu.mit.keycloak.util;
+
+public class OpenAttribute {
+  public static final String HOME_URL = "mitOpenHomeUrl";
```

Comment:
Right now we're using MIT open as the URL.  I recognize there's also a high likely-hood that we never change that.  But do you think we should assign a more generic name to this attribute and associated functionality?  For example, renaming it to "special redirect URL" would seem to provide more context for the variable's functionality.